sunpy/coordinates/ephemeris.py

```diff
@@ -216,8 +216,13 @@ def get_horizons_coord(body, time='now', id_type='majorbody', *, include_velocit
         If 'majorbody', search by name for planets, satellites, or other major bodies.
         If 'smallbody', search by name for asteroids or comets.
         If 'id', search by ID number.
-    time : {parse_time_types}
-        Time to use in a parse_time-compatible format
+    time : {parse_time_types}, `dict`
+        Time to use in a parse_time-compatible format.
+
+        Alternatively, this can be a dictionary defining a range of times and
+        dates; the range dictionary has to be of the form
+        {{'start':’YYYY-MM-DD [HH:MM:SS]’, 'stop':’YYYY-MM-DD [HH:MM:SS]’, 'step':’n[y|d|m|s]’}}.
```

Comment:
Just coming back to this now, and thought that instead argument could be a list of `[Time, Time, int]` which we internally convert to the jplhorizons string format. Any thoughts @ayshih?